shentu-2.2/distribution/README.md

```diff
@@ -0,0 +1,9 @@
+
+
+### Estimated distribution time
+- September 27, 2023, 13:00 UTC
+
+### Distribution rewards through DeepWallet
+- Login [DeepWallet](https://wallet.shentu.technology/login).
```

Comment:
Would it be better to mark the steps with numbers? like Step-by-Step Guide: 1. 2. 3.